Geometry/VeryForwardGeometryBuilder/interface/DetGeomDesc.h

```diff
@@ -30,62 +36,88 @@ class CTPPSRPAlignmentCorrectionData;
  \verbatim
     x_g = rotation * x_l + translation
  \endverbatim
+ *
+ * July 2020: Migrated to DD4hep
+ * To avoid any regression with values from XMLs / Geant4, all lengths are converted from cm (DD4hep) to mm. 
+ *
  **/
 
+struct DiamondDimensions {
+  double xHalfWidth;
+  double yHalfWidth;
+  double zHalfWidth;
+};
+
 class DetGeomDesc {
 public:
   using Container = std::vector<DetGeomDesc*>;
   using RotationMatrix = ROOT::Math::Rotation3D;
   using Translation = ROOT::Math::DisplacementVector3D<ROOT::Math::Cartesian3D<double>>;
```

Comment:
Is this the same `Translation` as `DDTranslation.h`?

Reply:
Yes the DD4hep translations are the ROOT::Math::XYZVector
(typedef for ROOT::Math::DisplacementVector3D<ROOT::Math::Cartesian3D<double> > )
In this specific case, ROOT was already used.